### ISSUE TITLE
configure fails: ./configure: 3314: Syntax error: redirection unexpected (expecting word)

### ISSUE THREAD

**Issue by yurivict** [astamm/nloptr#29]
```
** package ‘nloptr’ successfully unpacked and MD5 sums checked
checking for g++... no
checking for c++... c++
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether c++ accepts -g... yes
checking how to run the C++ preprocessor... c++ -E
checking whether we are using the GNU C++ compiler... (cached) yes
checking whether c++ accepts -g... (cached) yes
checking for pkg-config... yes
configure: Now testing for NLopt header file.
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking nlopt.h usability... no
checking nlopt.h presence... no
checking for nlopt.h... no
./configure: 3314: Syntax error: redirection unexpected (expecting word)
./configure: 3313: Syntax error: Error in command substitution
ERROR: configuration failed for package ‘nloptr’
* removing ‘/usr/home/yuri/R/amd64-portbld-freebsd11.0-library/3.4/nloptr’
Warning in install.packages :
  installation of package ‘nloptr’ had non-zero exit status
```

Found on the FreeBSD 11

**Comment by yurivict** [comment]
Possibly related to https://github.com/jyypma/nloptr/issues/20

**Comment by yurivict** [comment]
https://github.com/jyypma/nloptr/pull/31

**Comment by yurivict** [comment]
@jyypma Could you, please, fix this bug? It still can't be built on FreeBSD.

**Comment by RCIIIcm** [comment]
** package 'nloptr' successfully unpacked and MD5 sums checked
    checking for g++... g++
    checking whether the C++ compiler works... yes
    checking for C++ compiler default output file name... a.out
    checking for suffix of executables...
    checking whether we are cross compiling... no
    checking for suffix of object files... o
    checking whether we are using the GNU C++ compiler... yes
    checking whether g++ accepts -g... yes
    checking how to run the C++ preprocessor... g++ -E
    checking whether we are using the GNU C++ compiler... (cached) yes
    checking whether g++ accepts -g... (cached) yes
    checking for pkg-config... yes
    configure: Now testing for NLopt header file.
    checking for grep that handles long lines and -e... /usr/bin/grep
    checking for egrep... /usr/bin/grep -E
    checking for ANSI C header files... yes
    checking for sys/types.h... yes
    checking for sys/stat.h... yes
    checking for stdlib.h... yes
    checking for string.h... yes
    checking for memory.h... yes
    checking for strings.h... yes
    checking for inttypes.h... yes
    checking for stdint.h... yes
    checking for unistd.h... yes
    checking nlopt.h usability... no
    checking nlopt.h presence... no
    checking for nlopt.h... no
    configure: Need to download and build NLopt
    trying URL 'http://ab-initio.mit.edu/nlopt/nlopt-2.4.2.tar.gz'
    Content type 'application/octet-stream' length 2361992 bytes (2.3 MB)
    ==================================================
    downloaded 2.3 MB
    
    configure: Starting to install library to /tmp/Rtmp0xak4H/R.INSTALL8196088ef93/nloptr/nlopt-2.4.2
    ./configure: syntax error: `< ' unexpected
    ERROR: configuration failed for package 'nloptr'
    * removing '/usr/local/lib/R/library/nloptr'

I think this is the same bug, OpenBSD 6.2.

Ah, I see you fixed this - the current version on Github installed just fine.

**Comment by astamm** [comment]
We just released `v2.0.0`. We now rely on `Cmake` when building from source. Can you test and tell us if it fixes whatever issue you had in the past please?

**Comment by yurivict** [comment]
2.0.0 installs fine now.

Thanks!
